src/app/tabs/newAdmin/textrecords/components/ViewCustomRecordsComponent.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import { multilanguage } from 'redux-multilanguage';
+import { Button } from 'react-bootstrap';
+import { DisplayTextRecordContainer } from '../containers';
+import { TEXT_RECORD } from '../../resolver/types';
+import UserWaitingComponent from '../../../../components/UserWaitingComponent';
+
+const ViewCustomRecordsComponent = ({
+  content, gettingContent,
+}) => {
+  if (gettingContent) {
+    return <UserWaitingComponent visible />;
+  }
+  const [isShown, setIsShown] = useState(false);
+  const handleClick = () => {
+    setIsShown(current => !current);
+  };
+
+  React.useEffect(() => {
+  });
+  const switchViewType = (item) => {
+    if (item[1].isEmpty) {
+      return <></>;
+    }
+    const eipKeys = ['email', 'url', 'avatar', 'description', 'notice', 'keywords', 'com.discord', 'com.github', 'com.reddit', 'com.twitter ', 'org.telegram'];
+    const customValuesKeys = item[1].value.filter(c => !eipKeys.includes(c.id));
+    switch (item[0]) {
+      case TEXT_RECORD:
+        return (
+          <>
+            <p>
+              <strong>CUSTOM KEYS</strong>
+              {'    '}
```

Comment:
Use CSS if you need to add padding/margin not spaces.

Reply:
Change addressed.
`className="keys"`
```
 .keys {
    padding-right: 20px;
  }
```